src/usecases/retrieveHospitalsByTrustId.js

```diff
@@ -1,4 +1,7 @@
-const retrieveHospitalsByTrustId = ({ getDb }) => async (trustId) => {
+const retrieveHospitalsByTrustId = ({ getDb }) => async (
+  trustId,
+  options = {}
```

Comment:
Could we set the default withWards value here? That way it's easy to see what options this usecase takes without digging into the rest of the code

Reply:
Ah yush!